sql/wsrep_trans_observer.h

```diff
@@ -378,6 +378,12 @@ static inline int wsrep_after_commit(THD *thd, bool all) {
 
   /* It has to already unregistered from wsrep_group_commit_queue either
   during SE commit or directly in ha_commit_low if SE commit was not done. */
+  if (thd->wsrep_enforce_group_commit) {
```

Comment:
I think this commit is not required now.

The original comment was this.
```
    /* Ideally, for one-phase (with binlog=off) or two-phase (with binlog=on)
    this step would be executed when transaction commits in InnoDB.
    If galera node is acting as async slave and replicated action from async
    master result in empty changes on slave (slave directly applied the said
    changes and has skipped error through skip-slave-error configuration) it
    can result in said situation. In this case slave protocol directly commits
    gtid through gtid_end_transaction that invokes ordered_commit causing
    thread handler to register in wsrep group commit queue but since storage
    engine commit is not done it would fail to unregister the said thread
    handler as part of storage engine commit. Handle unregistration here. */
```
Can you restore this comment if it is still valid now?

Reply:
The original comment is not relevant in this place. It was moved to `ha_commit_low`
I updated the current comment.